routers/repo/issue.go

```diff
@@ -1549,14 +1547,12 @@ func ChangeCommentReaction(ctx *context.Context, form auth.ReactionForm) {
 
 	switch ctx.Params(":action") {
 	case "react":
-		if !util.IsStringInSlice(form.Content, setting.UI.Reactions) {
```

Comment:
Why remove these line?

Reply:
because of https://github.com/go-gitea/gitea/pull/9220/files#diff-61ce589b0f29ee34fc2448fcda3af467R100-R103

Reply:
and I added an Error Type for that see below

Reply:
@lunny can I resolve?